model_demos/requirements.txt

```diff
@@ -9,6 +9,5 @@ soundfile==0.12.1  # For Whisper
 librosa==0.10.0  # For Whisper
 numba==0.53.1  # For Whisper
 segmentation-models-pytorch==0.3.3  # For U-Net
-pylocron==0.2.1  # For U-Net
 diffusers==0.14.0  # For Stable Diffusion
 transformers==4.35.2 # For Fuyu8B
```

Comment:
Remove `transformers==4.35.2 # For Fuyu8B` as it's already this version in the pybuda whl.

Reply:
done here https://github.com/tenstorrent/tt-buda-demos/pull/28/commits/33e5aef3a2ace36d92177d3a7b0458fdc4d9acfe